src/sage/rings/padics/padic_generic_element.pyx

```diff
@@ -4036,8 +4049,8 @@ cdef class pAdicGenericElement(LocalGenericElement):
             return Rational(K.prime())**(-self.valuation())
 
     cpdef bint _is_base_elt(self, p) except -1:
-        """
-        Return ``True`` if this element is an element of Zp or Qp (rather than
+        r"""
+        Return ``True`` if this element is an element of `\ZZ_p` or `\QQ_p` (rather than
         an extension).
```

Comment:
Here you rewrite `Zp` and `Qp` as single-quoted`\ZZ_p` and `\QQ_p`. In other places, You write them as double-quoted `Zp` and `Qp`. Are these different cases, or you are just inconsistent?